docs/dev_guide/diagnostic_ref/dpct1078.rst

```diff
@@ -24,4 +24,42 @@ Suggestions to Fix
 ------------------
 
 Replace ``memory_order::acq_rel`` with ``memory_order::seq_cst`` if stricter
-memory order is needed.
\ No newline at end of file
+memory order is needed.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   __device__ void foo_dev() {
+     ...
+     __threadfence();
+     ...
+   }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo_dev() {
+     ...
+     /*
+     DPCT1078:0: Consider replacing memory_order::acq_rel with
+     memory_order::seq_cst for correctness if strong memory order restrictions are
+     needed.
+     */
+     sycl::atomic_fence(sycl::memory_order::acq_rel, sycl::memory_scope::device);
+     ...
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo_dev() {
+     ...
```

Comment:
Add a comment in L62, which require strong memory order.

Reply:
done